src/pages/SignUpSecond.js

```diff
@@ -0,0 +1,140 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './SignUpSecond.scss';
+
+const SignUpSecond = () => {
+  const [inputValue, setInputValue] = useState({});
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  let emailRegular =
+    /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i;
+
+  let pwRegular = /^(?=.*[a-zA-Z])(?=.*[!@#$%^*+=-])(?=.*[0-9]).{10,16}$/;
+
+  const emailCondition = emailRegular.test(inputValue.email);
+  const pwCondition = pwRegular.test(inputValue.pw);
+  const pwCheckCondition = inputValue.pwCheck === inputValue.pw;
+  const nameCondition = inputValue.name?.length > 0;
+  const phoneCondition = inputValue.phone?.length === 11;
+
+  const signUpCondition =
+    emailCondition &&
+    pwCondition &&
+    pwCheckCondition &&
+    nameCondition &&
+    phoneCondition;
+
+  const navigate = useNavigate();
+
+  const handleInput = e => {
+    const { name, value } = e.target;
+    setInputValue({ ...inputValue, [name]: value });
+  };
+
+  const postUserSignUpData = () => {
+    fetch('http://10.58.52.88:3000/users/signup', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        email: inputValue.email,
+        password: inputValue.pw,
+        name: inputValue.name,
+        phoneNumber: inputValue.phone,
+      }),
```

Comment:
Input에 name 속성을 조금 융통성 있게 백엔드에서 요구하는 키값과 동일하게 관리하면
```suggestion
      body: JSON.stringify(inputValue),
```
로 줄일 수 있을 것 같습니다.
[간단한 예시](https://velog.io/@e_juhee/shorthand-property#:~:text=Shorthand%20Property,-%EB%8B%A8%EC%B6%95%20%EC%86%8D%EC%84%B1%EB%AA%85&text=%EA%B0%9D%EC%B2%B4%EB%A5%BC%20%EC%A0%95%EC%9D%98%ED%95%A0%20%EB%95%8C,%EC%95%8A%EA%B3%A0%20%ED%95%9C%EB%B2%88%EB%A7%8C%20%ED%91%9C%EA%B8%B0%ED%95%B4%EB%8F%84%20%EB%90%9C%EB%8B%A4!) 한번 확인해보세요